fixtures/appleTrendsListenersByEpisode.json

```diff
@@ -0,0 +1,985 @@
+{
+  "provider": "apple",
+  "version": 1,
+  "retrieved": "2022-10-04T10:09:46.463Z",
+  "meta": {
+    "show": "1603082924",
+    "endpoint": "showTrends/Listeners"
```

Comment:
Interesting idea with the slash here. I wonder if `showTrends` is really relevant. Alternatively we could just call it `listeners` because the source (the route) of the data might be irrelevant. What do you think?

Reply:
I think it is good to see the api name for debugging, otherwise you have to look up the mapping somewhere